osdk/src/base_crate/Cargo.toml.template

```diff
@@ -2,3 +2,7 @@
 name = "#NAME#"
 version = "#VERSION#"
 edition = "2021"
+
+[dependencies]
+ostd = { version = "0.8.0", path = "../../../ostd" }
```

Comment:
Don't hard code the dependency here. Add them by OSDK. The versions need to match the OSDK version.

Reply:
Use `env!(CARGO_PACKAGE_VERSION)` when generating the dependency. You can find similar implementations in OSDK.